config/jobs/kubernetes/sig-node/dra-ci.yaml

```diff
@@ -106,19 +98,19 @@ periodics:
             cpu: 2
             memory: 9Gi
 
-  # This job runs e2e_node.test with a focus on tests for the Dynamic Resource Allocation feature (currently beta)
   - name: ci-node-e2e-cgrpv1-crio-dra
     cluster: k8s-infra-prow-build
     interval: 6h
-    annotations:
-      testgrid-dashboards: sig-node-cri-o, sig-node-dynamic-resource-allocation
-      testgrid-tab-name: ci-node-e2e-cgrpv1-crio-dra
-      description: Runs E2E node tests for Dynamic Resource Allocation beta features with CRI-O using cgroup v1
-      testgrid-alert-email: eduard.bartosh@intel.com,patrick.ohly@intel.com
-      fork-per-release: "true"
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
```

Comment:
This adds `preset-pull-kubernetes-e2e` and `preset-pull-kubernetes-e2e-gce` to `ci-node-e2e-cgrpv1-crio-dra` which didn't have them before.

Is that necessary? If not, let's simplify the job.

Reply:
Not necessary, but simplifies the template.
Removed from everywhere, but that can cause some pull and canary job failures. will see.

Reply:
On the master branch, `pull-kubernetes-node-e2e-crio-cgrpv1-dra` had those presets, `ci-node-e2e-cgrpv1-crio-dra` didn't. Do you know what those presets do? I don't.

Let's try without them.

Reply:
Clarified on Slack: those presets don't do much and shouldn't be needed here.